quirks.bs

```diff
@@ -240,6 +240,8 @@ above properties in its "Value" grammar.
 supported in the <code><a method for=CSS lt="supports(property, value)">supports()</a></code> static
 method of the {{CSS}} interface.</p>
 
+<p><a>Quirky colors</a> must not be supported on a <a interface>CSSStyleSheet</a> where the <a>constructed flag</a> is set.</p>
```

Comment:
Use `{{CSSStyleSheet}}` instead, to get the proper monospace font. Here and below.